.github/workflows/main.yml

```diff
@@ -61,7 +62,13 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: helm-artifacts
-          path: ${{ github.workspace }}/helm-charts
+          path: ./helm-charts
+
+      - name: Upload cluster configs
+        uses: actions/upload-artifact@v4
+        with:
+          name: cluster-artifact
+          path: ./cluster.yml
```

Comment:
Verify that the path `./cluster.yml` is correct and that the cluster configuration file is available at this location during the workflow execution. This is essential for uploading the cluster configuration.